### ISSUE TITLE
This file does not have an app associated with it... create an association in the Default Apps Settings page.

### ISSUE THREAD

**Issue by StayBlue** [rcmaehl/MSEdgeRedirect#345]
### Preflight Checklist

- [ ] Running `microsoft-edge:https://google.com` redirects successfully
- [ ] Microsoft Edge is selected in any "How do you want to open this?" box

### Install Type

New Install

### Install Mode

Active Mode

### Steps to reproduce

Attempt to open google.com with Win+R.

It doesn't show any logs, unfortunately.

### ✔️ Expected Behavior

For MSEdgeRedirect to work and redirect me to my default browser

### ❌ Actual Behavior

It gave an error
![image](https://github.com/rcmaehl/MSEdgeRedirect/assets/23127866/da62f99d-6916-49b7-bb8b-ee7a659822b5)

I tried the latest test build and the latest stable version, both to no avail.

### Microsoft Windows version

22H@ Build 22621.2715

### Other Software

I have Microsoft Edge uninstalled on my system.

**Comment by rcmaehl** [comment]
Can you verify if you used AveYo's edge removal tool or another script that uses it? There is a specific file needed to be left behind for MSEdgeRedirect to work

**Comment by StayBlue** [comment]
> Can you verify if you used AveYo's edge removal tool or another script that uses it? There is a specific file needed to be left behind for MSEdgeRedirect to work

I think I used AveYo's edge removal tool when I removed Edge, but I don't recall for sure. Thanks for the very fast reply.

**Comment by StayBlue** [comment]
![image](https://github.com/rcmaehl/MSEdgeRedirect/assets/23127866/0fa823a5-630a-452c-b703-4bb91d7f34b5)

I think this confirms that I used AveYo's edge removal tool.

**Comment by rcmaehl** [comment]
Odd. I'll look into this. Can you verify you have `ie_to_edge_stub.exe` in either:

- C:\Scripts
- C:\ProgramData
- C:\Users\Public

**Comment by StayBlue** [comment]
> Odd. I'll look into this. Can you verify you have `ie_to_edge_stub.exe` in either:
> 
>     * C:\Scripts
> 
>     * C:\ProgramData
> 
>     * C:\Users\Public

I have it in `C:\Scripts`

**Comment by rcmaehl** [comment]
@aveyo Any idea?

**Comment by StayBlue** [comment]
I fixed the Microsoft Edge redirect not working by re-installing Microsoft Edge from Microsoft's website and using AveYo's Edge removal tool again. However, I'm now getting a different error saying invalid parameters along with the link it's trying to open when I try to open a link in an app.

I could open a separate issue for this if you'd like.

**Comment by rcmaehl** [comment]
> I fixed the Microsoft Edge redirect not working by re-installing Microsoft Edge from Microsoft's website and using AveYo's Edge removal tool again. However, I'm now getting a different error saying invalid parameters along with the link it's trying to open when I try to open a link in an app.
> 
> I could open a separate issue for this if you'd like.

Nope. Let me release 0.7.5.1

**Comment by StayBlue** [comment]
> > I fixed the Microsoft Edge redirect not working by re-installing Microsoft Edge from Microsoft's website and using AveYo's Edge removal tool again. However, I'm now getting a different error saying invalid parameters along with the link it's trying to open when I try to open a link in an app.
> 
> > 
> 
> > I could open a separate issue for this if you'd like.
> 
> 
> 
> Nope. Let me release 0.7.5.1

Okay, thanks. Just to let you know, this is happening on the latest test build.

**Comment by rcmaehl** [comment]
> > > I fixed the Microsoft Edge redirect not working by re-installing Microsoft Edge from Microsoft's website and using AveYo's Edge removal tool again. However, I'm now getting a different error saying invalid parameters along with the link it's trying to open when I try to open a link in an app.
> > 
> > 
> > > 
> > 
> > 
> > > I could open a separate issue for this if you'd like.
> > 
> > 
> > Nope. Let me release 0.7.5.1
> 
> Okay, thanks. Just to let you know, this is happening on the latest test build.

Can you post a screenshot of the error please?

**Comment by rcmaehl** [comment]
Odd. Thank you. Looking into this right now

**Comment by rcmaehl** [comment]
Can you verify when you re-removed edge it isn't still et as the default browser?

**Comment by rcmaehl** [comment]
Let me rephrase that. Can you check in Default Apps that something other than Edge is set as the Default Browser?

For some reason the IFEO check is thinking it's not being passed data from ie_to_edge_stub.exe

**Comment by StayBlue** [comment]
> Let me rephrase that. Can you check in Default Apps that something other than Edge is set as the Default Browser?
> 
> For some reason the IFEO check is thinking it's not being passed data from ie_to_edge_stub.exe

Firefox thinks that my default browser is set to it in its settings.
![image](https://github.com/rcmaehl/MSEdgeRedirect/assets/23127866/0093826f-a408-4c11-8665-cfbbe00ac001)

Additionally, I pressed the button below and re-ran the test before, and I still got the same error.
![image](https://github.com/rcmaehl/MSEdgeRedirect/assets/23127866/5b4bb426-86f9-447c-a59b-2d1eb510345b)

The reason this is happening is also beyond me.

**Comment by rcmaehl** [comment]
Please try the newest test build here in about 2-3 minutes

**Comment by StayBlue** [comment]
> Please try the newest test build here in about 2-3 minutes

I still get the exact same error message 😭

**Comment by FadeMind** [comment]
@rcmaehl After uninstall Edge in system Edge Protocol  does not exists anymore, so MSER  does not work anymore. Reinstall Edge = Applying MSER = redirect works. Workaround? Add fake `MICROSOFT-EDGE` protocol IF Edge was removed from system.

**Comment by rcmaehl** [comment]
> @rcmaehl After uninstall Edge in system Edge Protocol does not exists anymore, so MSER does not work anymore. Reinstall Edge = Applying MSER = redirect works. Workaround? Add fake `MICROSOFT-EDGE` protocol IF Edge was removed from system.

This didn't use to be the case. Maybe it's due to the recent EEA changes. I'll double check this add that in if that turns out to be the case!

**Comment by rcmaehl** [comment]
I said "ATTEMPT TO FIX" github. Not "FIXED"

**Comment by rcmaehl** [comment]
I've readded the microsoft-edge: association. Please try 0.7.5.2 when you have a chance

**Comment by FadeMind** [comment]
@rcmaehl 
![20231222-1703256911](https://github.com/rcmaehl/MSEdgeRedirect/assets/9846948/61bb2ab4-f189-42d4-a5d7-acb7d2d07175)

**Comment by rcmaehl** [comment]
> @rcmaehl ![20231222-1703256911](https://private-user-images.githubusercontent.com/9846948/292513774-61bb2ab4-f189-42d4-a5d7-acb7d2d07175.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMyNTcyOTYsIm5iZiI6MTcwMzI1Njk5NiwicGF0aCI6Ii85ODQ2OTQ4LzI5MjUxMzc3NC02MWJiMmFiNC1mMTg5LTQyZDQtYTVkNy1hY2I3ZDJkMDcxNzUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMjJUMTQ1NjM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTliNjcyMjhjODY4YjZjNmRmNDUxZDYwMjVmOTI2ODBlYWFkMTljNWNmOTIxY2Y5MDdiYzFiYzU1MGQzMDJkYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.fT9-JS2C_3MeK_dL6ZQzlpyI4DhplRSqltCvuKjfJyQ)

Are you using the test build or 0.7.5.1?

**Comment by rcmaehl** [comment]
> @rcmaehl ![20231222-1703256911](https://private-user-images.githubusercontent.com/9846948/292513774-61bb2ab4-f189-42d4-a5d7-acb7d2d07175.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMyNTczMzYsIm5iZiI6MTcwMzI1NzAzNiwicGF0aCI6Ii85ODQ2OTQ4LzI5MjUxMzc3NC02MWJiMmFiNC1mMTg5LTQyZDQtYTVkNy1hY2I3ZDJkMDcxNzUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMjJUMTQ1NzE2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWE1YmE0NWRjMWZlYzI3ZjVlMjg0OThmM2EzMGJlYjE5ZjU4Mzc0Y2MxMjgyMDhhNTk3OTllYzExZWMxMjQ2OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.yDE1bld-PtWZY1H3ANxxKi0_dhQPgBNT-jHHibOfoKo)

From the just released 0.7.5.2?

**Comment by FadeMind** [comment]
@rcmaehl 
I have removed Edge already via > https://github.com/ShadowWhisperer/Remove-MS-Edge
I have reinstalled WebView2 from online installer.
After running MSEdgeRedirect latest version does  nothing change. 

I can share logs, reinstall standalone edge from web installer if you like.

**Comment by FadeMind** [comment]
See also > https://github.com/ShadowWhisperer/Remove-MS-Edge/blob/main/_Source/remove.py#L65

**Comment by rcmaehl** [comment]
This should be patched with #519
